python/chronos/script/benchmark-chronos.py

```diff
@@ -15,153 +15,253 @@
 #
```

Comment:
not quiet sure why this file's permission code is changed

Reply:
has changed back